packages/formspree-react/tsconfig.json

```diff
@@ -1,17 +1,15 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "@formspree/react",
+  "extends": ["../../tsconfig.json"],
   "compilerOptions": {
     "target": "es5",
```

Comment:
This should be removed (i.e. to use root `tsconfig` value for consistency between core and react packages) in the future. Since it will produce a very different built `index.js`, we will update it as part of a breaking change release.

cc/ @colevscode